.github/workflows/lint-php.yml

```diff
@@ -2,10 +2,26 @@ name: Lint PHP
 
 on:
   push:
+    # Only run if PHP-related files changed.
+    paths:
+      - '**.php'
```

Comment:
Would this cover `scoper.inc.php`?

Reply:
[Yes.](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2A%2A.php&matches=false&tests=scoper.inc.php)